go/teams/chain.go

```diff
@@ -249,6 +249,21 @@ func (t TeamSigChainState) GetUsersWithRole(role keybase1.TeamRole) (res []keyba
 	return res, nil
 }
 
+func (t TeamSigChainState) GetLatestUVWithUID(uid keybase1.UID) (res keybase1.UserVersion, err error) {
+	found := false
+	for uv := range t.inner.UserLog {
+		if uv.Uid == uid && (!found || res.EldestSeqno < uv.EldestSeqno) {
```

Comment:
Might want to only do match for people still in the team. The userlog includes removals. So add `&& *t.GetUserLogPoint(uv).Role != keybase1.TeamRole_NONE`